pyhtools/attackers/web/email_scraper.py

```diff
@@ -0,0 +1,74 @@
+from bs4 import BeautifulSoup
+import requests, re, lxml
+from requests import exceptions
+from urllib import parse
+from collections import deque
+from pyhtools.UI.colors import BRIGHT_RED, BRIGHT_WHITE, BRIGHT_YELLOW
```

Comment:
python std library imports should be first followed by third party libraries and pyhtools import at the last.